keyboards/adkb96/rev1/config.h

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+Copyright 2015 Jack Humbert
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x00a5
+#define PRODUCT_ID      0xad96
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Bit Trade One
+#define PRODUCT         ADKB96
+#define DESCRIPTION     
+
+/* key matrix size */
+// Rows are doubled-up
+#define MATRIX_ROWS 12
+#define MATRIX_COLS 8
+
+// wiring of each half
+#define MATRIX_ROW_PINS { D4, C6, D7, E6, B4, B5 }
+#define MATRIX_COL_PINS { B6, B2, B3, B1, F7, F6, F5, F4 }
+
+#define SPLIT_HAND_PIN D0
```

Comment:
Delete SPLIT_HAND_PIN.

Reply:
Sorry, this should actually be: 
```suggestion
#define SOFT_SERIAL_PIN D0
```